modules/getting-started-openshift-common-terms.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-overview.adoc
+
+:_content-type: REFERENCE
+[id="getting-started-openshift-common-terms_{context}"]
+= Common Terms
+
+The following are common terms that are used within the Getting Started. These terms help you orient yourself with the content and other parts of the documentation.
+
+Kubernetes::
+Kubernetes is an open source container orchestration engine for automating deployment, scaling, and management of containerized applications.
+
+Containers::
+Containers are application instances and components that run in OCI-compliant containers on the worker nodes. A container is the runtime of an Open Container Initiative (OCI)-compliant image. An image is a binary application. A worker node can run many containers. A node capacity is related to memory and CPU capabilities of the underlying resources whether they are cloud, hardware, or virtualized.
+
+Pod::
+A pod is one or more containers deployed together on one host. It consists of a colocated group of containers with shared resources such as volumes and IP addresses. A pod is also the smallest compute unit defined, deployed, and managed.
++
+In {product-title}, pods replace individual application containers as the smallest deployable unit.
++
+Pods are the orchestrated unit in {product-title}. {product-title} schedules and runs all containers in a pod on the same node. Complex applications are made up of many pods, each with their own containers. They interact externally and also with another inside the {product-title} environment. {product-title} runs container images in containers wrapped by a meta object called "pod".  It is possible to have multiple containers in a single pod. For example, to support cluster features you can have a "sidecar" container.
+
+ReplicaSet and ReplicationController::
+The Kubernetes ReplicaSet and the {product-title} replication controller are both available. The job of this component is to ensure the specified number of pod replicas are running at all times. If pods exit or are deleted, the ReplicaSet or replication controller starts more. If more pods are running than needed, ReplicaSet deletes as many as necessary to match the specified number of replicas.
+
+Deployment and DeploymentConfig::
+{product-title} implements both Kubernetes deployments and {product-title} DeploymentConfigs. Users may select either.
```

Comment:
either say "deployments" and "deployment configs" or say "`Deployment obejcts`" and "`DeploymentConfig` objects" (throughout module)

Technically "may" might be fine here since it's kind of permission, but I'd suggest "can" instead of you think it works